test/tests/CMakeLists.txt

```diff
@@ -188,3 +188,10 @@ set(TILE_ELEMENT_TEST_SOURCES "${CMAKE_CURRENT_LIST_DIR}/TileElements.cpp"
 add_executable(test_tile_elements ${TILE_ELEMENT_TEST_SOURCES})
 target_link_libraries(test_tile_elements ${GTEST_LIBRARIES} libopenrct2 ${LDL} z)
 add_test(NAME tile_elements COMMAND test_tile_elements)
+
+# Pathfinding test
+set(PATHFINDING_TEST_SOURCES  "${CMAKE_CURRENT_LIST_DIR}/Pathfinding.cpp"
+                              "${CMAKE_CURRENT_LIST_DIR}/TestData.cpp")
+add_executable(test_pathfinding ${PATHFINDING_TEST_SOURCES})
+target_link_libraries(test_pathfinding ${GTEST_LIBRARIES} libopenrct2 ${LDL} z)
+add_test(NAME pathfinding COMMAND test_pathfinding)
```

Comment:
I see that this matches other setup in the file. What's it actually for?

Reply:
As discussed in Gitter - it's used for project file generation. I didn't see it before because I'm using VS, which has separate (versioned) project files.